<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>SongPB</class>
    <widget class="QWidget" name="SongPB">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>737</width>
                <height>74</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>737</width>
                <height>74</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>737</width>
                <height>74</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <widget class="QPushButton" name="pb_perf">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>20</y>
                    <width>121</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Segoe UI Black</family>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true"/>
            </property>
            <property name="text">
                <string/>
            </property>
        </widget>
        <widget class="QPushButton" name="pb_name">
            <property name="geometry">
                <rect>
                    <x>140</x>
                    <y>20</y>
                    <width>481</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Segoe UI Black</family>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true"/>
            </property>
            <property name="text">
                <string/>
            </property>
        </widget>
        <widget class="QPushButton" name="pb_like">
            <property name="geometry">
                <rect>
                    <x>630</x>
                    <y>10</y>
                    <width>51</width>
                    <height>51</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true"/>
            </property>
            <property name="text">
                <string>+</string>
            </property>
        </widget>
        <widget class="QPushButton" name="pb_add">
            <property name="geometry">
                <rect>
                    <x>680</x>
                    <y>10</y>
                    <width>51</width>
                    <height>51</height>
                </rect>
            </property>
            <property name="contextMenuPolicy">
                <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="styleSheet">
                <string notr="true"/>
            </property>
            <property name="text">
                <string>...</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
